shared/src/lib/dialects/mysql.ts

```diff
@@ -12,7 +12,9 @@ import {
 
 const types = [
   ...SpecialTypes,
-  'bit', 'int', 'integer', 'tinyint', 'smallint', 'mediumint', 'bigint', 'float', 'double', 'double precision', 'dec', 'decimal', 'numeric', 'fixed', 'bool', 'boolean', 'date', 'datetime', 'timestamp', 'time', 'year', 'char', 'nchar', 'national char', 'varchar', 'nvarchar', 'national varchar', 'text', 'tinytext', 'mediumtext', 'blob', 'longtext', 'tinyblob', 'mediumblob', 'longblob', 'enum', 'set', 'json', 'binary', 'varbinary', 'geometry', 'point', 'linestring', 'polygon', 'multipoint', 'multilinestring', 'multipolygon', 'geometrycollection'
+  'bit', 'int', 'int unsigned', 'integer', 'integer unsigned', 'tinyint', 'tinyint unsigned',
```

Comment:
When we query a table for types, do we appropriately bring back the `unsigned` component? Like if you load a table in the TableStructure tab, does it show unsigned integers?

Can you add a test to the mysql integration tests to make sure that you can both create, and retrieve unsigned columns.

Reply:
Fortunately, it just shows the `unsigned`. Pretty handy! Also in the main table view. And I'm adding the tests now.

![image](https://github.com/beekeeper-studio/beekeeper-studio/assets/38707148/b2d42e9a-0e51-4dba-8bfc-980521a3768d)